.env.example

```
@@ -59,5 +59,6 @@ STATSIG_API_KEY=
 
 # Other environment variables
 NODE_ENV=development
-PORT=8080
+PORT=5000
+CORS_ORIGINS="http://localhost:3000,http://localhost:4000,http://localhost:3001"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)